cluster-operator/src/test/java/io/strimzi/operator/cluster/model/AbstractModelTest.java

```diff
@@ -63,4 +64,40 @@ public void testJvmMemoryOptionsMemoryRequest() {
         assertEquals("0.7", env.get(AbstractModel.ENV_VAR_DYNAMIC_HEAP_FRACTION));
         assertEquals("10000000000", env.get(AbstractModel.ENV_VAR_DYNAMIC_HEAP_MAX));
     }
+
+    @Test
+    public void testJvmPerformanceOptions() {
+        JvmOptions opts = JvmOptions.fromJson("{}");
+
+        assertNull(getPerformanceOptions(opts));
+
+        opts = JvmOptions.fromJson("{\n" +
+                "  \"-server\": \"true\"\n" +
+                "}");
```

Comment:
why not just one line for this simple JSON ?

Reply:
Might be subjective, but for me it is much easier to read and edit like this. The only place where I used one line JSON was when adding something to existing one line JSONs.

Reply:
Yeah it's subjective but maybe you don't really need the "\n" which just add noise. Without "\n" is should be still simple to read for you from a developer perspective I guess. Btw up to you.